components/tree-select/index.en-US.md

```diff
@@ -0,0 +1,56 @@
+---
+category: Components
+type: Form Controls
+title: TreeSelect
+---
+
+Tree selection control.
+
+## When to use
+
+`TreeSelect` is a selection control like `Select`, which has a tree shaped data structure, such as corporate hierarchy、 disciplinary system、 directory structure and etc.
```

Comment:
For the rest of tree-select / index.en-US.md, I would provide some adjustments after the commit to master as there are many entries I would like to change: for example, I would make all the description start with capitals. 😄